src/shadowbox/model/access_key.ts

```diff
@@ -48,9 +48,16 @@ export interface AccessKey {
   readonly dataLimit?: DataLimit;
 }
 
+export interface AccessKeyCreateParams {
+  // The unique identifier to give the access key. Throws if it exists.
+  readonly id?: AccessKeyId;
```

Comment:
Perhaps we should omit this, and only allow setting the id via the PUT call. 
Otherwise this field will be ambiguous on PUT calls. Which one takes precedence? I'm strongly opposed to allowing ids to be changed, since that goes against the semantics of identity.